### DISCUSSION TITLE
AI的未來

### DISCUSSION THREAD

**Discussion by Rodrigo-dev-bit** [1101-GS4719-C/Python-Community-GS4719-C#802] [Ideas]
這幾天我看到這一篇[文章](https://www.newscientist.com/article/2342632-artificial-intelligence-is-being-asked-to-predict-the-future-of-ai/)，內容就是用AI來推測AI的發展，我很好奇就繼續查一些資料，然後查到之前有一個德國研究把1994年的科技資料給AI推測我們現在的發展，結果這個AI的推測99%都對，所以我覺得以後我們生活上的問題都可能被AI給處理掉，你們大家對AI的未來有什麼看法呢？

**Comment by moonnight3503** [comment]
感覺未來會有多數產業被AI取代，AI越來越先進，已經越來越接近人類的大腦

**Comment by Pietra1226** [comment]
我認為若給AI他們自由發揮的空間其實他們已經超越人腦了，不斷的training而且他們不用像人類一樣需要休息。

**Comment by eason8479** [comment]
我個人認為ai在解決問題方面已經超出人類，但是人類在提出問題方面仍有不可取代的作用

舉個例子，最近很火的繪圖ai
哪怕ai在繪圖領域在怎麼強（解決問題），
但製作圖片的那串指令還是要由人類輸入（提出問題）
更廣義來說，如何利用ai來快速繪圖這件事也是由人類提出的問題

**Comment by evanchou1204** [comment]
當一切預測都100%準確的時候，可能就是命定論被證明的時候了

**Comment by WEI-CHEN-CHUN** [comment]
畢竟AI可以短時間重複計算模擬非常多次，人類在這方面是不可能超越，但思考、發想目前仍是AI無法達到的部分

**Comment by Chiway1229** [comment]
我覺得ai勢必成為未來時代發展的趨勢，ai演算普遍化是遲早會發生的

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
如果給AI越來越越多的權限 相信再不久的將來 它一定會超越人類 在來就是我們人能還能做什麼呢

**Comment by StevenWanglolz** [comment]
感覺在設計AI上要加上層層限制，不然很有可能會像一匹脫韁的野馬，一發不可收拾

**Comment by swerfg03joe** [comment]
AI也是分很多種的，首先他的思考方式是否與人類相同或是有自己的想法都很難去證明，因AI有一種被稱為監督式學習的訓練方式。它需要大量的訓練資料，也就是人類已知的資料，例如要做AI繪圖，他就需要送入大量的繪師作品讓他學習，講是講學習但實際上就是模仿，很難真的作出全新的風格。

**Comment by orianna901001** [comment]
我覺得ai確實是會越來越進步，但目前對於ai的進步，人類的提問或是訓練是不可缺少的，要進步到怎麼樣的境地，基於倫理問題我們也需要好好把關。
